golem-rib/regression_tests/lib.rs

```diff
@@ -15,108 +15,108 @@ use rib::{
 #[test]
 async fn test_rib_regression() {
     let expr = r#"
-
+              let worker = instance();
```

Comment:
This is now mandatory. If a component / worker needs to be used in rib script, there needs to be an instance.